R/visualization/README.md

```diff
@@ -0,0 +1,47 @@
+# Plotting Summary Tables - how to
```

Comment:
I think this is pretty good, people should be able to reuse the functions from this. I have some minor wording/formatting comments if you want but not super important. I think the main change is to make sure all the files/functions are covered. I can add something for mine but might be better to wait until @martaint's are finished to avoid conflicts. 

If you wanted to go a step further you could document the functions themselves with roxygen comments (see my files for examples) but probably not necessary.

Reply:
Oh now I found your comments! Sorry for the delay...great, thanks, then I will go on in the same way. Indeed, I used the roxygen format to add comments to the functions, gonna update them soon. please feel free to add your wording/formatting comments!